attributes/default.rb

```diff
@@ -58,6 +58,19 @@
 default['cfncluster']['intelmpi']['version'] = '2019.7.217'
 default['cfncluster']['intelmpi']['modulefile'] = "/opt/intel/impi/#{node['cfncluster']['intelmpi']['version']}/intel64/modulefiles/mpi"
 
+# Arm Performance Library
+default['cfncluster']['armpl']['version'] = '20.2.1'
+default['cfncluster']['armpl']['gcc']['major_minor_version'] = '9.3'
+default['cfncluster']['armpl']['gcc']['patch_version'] = '0'
```

Comment:
it looks like most of the times you need a concatenation between major/minor and patch version. I'd rather have a single attribute for the full version and extract the patch with a split when necessary. This way it's also easier to see the full used version in a single string.

Reply:
I am thinking if we can split string in erb file or not. We need to pass 9.3 and 9.3.0 to two erb files, if we use full version like  `default['cfncluster']['armpl']['gcc']['version'] = 9.3.0`, do you know how to deal with the erb files?
Or we can use two versions in default file, like `default['cfncluster']['armpl']['gcc']['gcc_version'] = 9.3.0`, `default['cfncluster']['armpl']['gcc']['armpl_version'] = 9.3`.